client/src/pages/Home.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useQuery } from 'react-query';
+import { loadProducts } from '../services/Product';
+import { ProductCard } from '../components/ProductCard';
+
+export function Home() {
+  const { data } = useQuery(['products'], loadProducts);
+
+  const handleClickItem = () => {};
+
+  const handleClickCart: React.MouseEventHandler<HTMLElement> = (event) => {
+    event.stopPropagation();
+  };
+
+  return (
+    <div className="py-16">
+      <ul className="container flex flex-wrap">
+        {data?.map((item) => (
+          <ProductCard
+            item={item}
+            onClickItem={handleClickItem}
+            onClickCart={handleClickCart}
+          />
```

Comment:
클릭 후 어떤 작업을 하는지 명시해주는 네이밍이면 좋지 않을까요? 
```suggestion
          <ProductCard
            item={item}
            onClickItem={아이템상세보기로이동한다}
            onClickCart={장바구니에추가한다}
          />
```

저는 이런 식으로 되면 좀 더 사용하는쪽에서 ProductCard 가 어떤 놈인지 파악하기 좋은 것 같아요!